content/running_bookwyrm/install-prod-dockerless.md

```diff
@@ -0,0 +1,147 @@
+- If you wish to use an external storage for static assets and media files (such as an S3-compatible service), [follow the instructions](/external-storage.html) until it tells you to come back here
+- Create and setup your `bookwyrm` user
+    - Make the system bookwyrm user:
+        `useradd bookwyrm -r`
+    - Change the owner of your install directory to bookwyrm:
+        `chown -R /opt/bookwyrm bookwyrm:bookwyrm`
+    - You should now run bookwyrm related commands as the bookwyrm user:
+        `sudo -u bookwyrm echo I am the $(whoami) user`
+
+- Generate the admin code with `sudo -u bookwyrm venv/bin/python3 manage.py admin_code`, and copy the admin code to use when you create your admin account.
+- You can get your code at any time by re-running that command. Here's an example output:
+
+``` { .sh }
+*******************************************
+Use this code to create your admin account:
+c6c35779-af3a-4091-b330-c026610920d6
+*******************************************
+```
+
+- Make and configure the run script
+    - Make a file called dockerless-run.sh and fill it with the following contents
+
+``` { .sh }
+#!/bin/bash
+
+# stop if one process fails
+set -e
+
+# bookwyrm
+/opt/bookwyrm/venv/bin/gunicorn bookwyrm.wsgi:application --bind 0.0.0.0:8000 &
+
+# celery
+/opt/bookwyrm/venv/bin/celery -A celerywyrm worker -l info -Q high_priority,medium_priority,low_priority &
+/opt/bookwyrm/venv/bin/celery -A celerywyrm beat -l INFO --scheduler django_celery_beat.schedulers:DatabaseScheduler &
+# /opt/bookwyrm/venv/bin/celery -A celerywyrm flower &
+```
+- 
+    - Replace `/opt/bookwyrm` with your install dir
+    - Change `8000` to your custom port number
+    - Flower has been disabled here because it is not autoconfigured with the password set in the `.env` file
+- You can now run BookWyrm with: `sudo -u bookwyrm bash /opt/bookwyrm/dockerless-run.sh`
+- The application should be running at your domain. When you load the domain, you should get a configuration page which confirms your instance settings, and a form to create an admin account. Use your admin code to register.
+- You may want to configure BookWyrm to autorun with a systemd service. Here is an example:
+```
+# /etc/systemd/system/bookwyrm.service
+[Unit]
+Description=Bookwyrm Server
+After=network.target
+After=systemd-user-sessions.service
+After=network-online.target
+
+[Service]
+User=bookwyrm
+Type=simple
+Restart=always
+ExecStart=/bin/bash /opt/bookwyrm/dockerless-run.sh
+WorkingDirectory=/opt/bookwyrm/
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Congrats! You did it!! Configure your instance however you'd like.
+
+## Get Connected
+
+Because BookWyrm is a young project, we're still working towards a stable release schedule, and there are a lot of bugs and breaking changes. There is a GitHub team which can be tagged when there's something important to know about an update, which you can join by sharing your GitHub username. There are a few ways in get in touch:
```

Comment:
The GitHub team bit is out of date -- I probably need to update the regular install instructions! Just linking to the "Get Involved" page would be a better bet here: https://joinbookwyrm.com/get-involved/